fuzz/Cargo.toml

```diff
@@ -10,6 +10,7 @@ cargo-fuzz = true
 
 [dependencies]
 arbitrary = { version = "1.4.1", features = ["derive"] }
+float-cmp = "0.10.0"
```

Comment:
`bvh` itself uses https://github.com/svenstaro/bvh/blob/ca417859cd3f33b129cd82079d1a90d7da543e0a/Cargo.toml#L18

```
abs_diff_eq!(1.0, 1.0);
abs_diff_eq!(1.0, 1.0, epsilon = f64::EPSILON);
```

Maybe the fuzzer should too, to avoid dependency proliferation.